addons/misra.py

```diff
@@ -1583,3 +1607,28 @@ def generateTable():
                 print('Not expected: ' + actual)
                 exitCode = 1
         sys.exit(exitCode)
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--rule-texts", type=str, help="Path to text file of MISRA rules")
+parser.add_argument("--suppress-rules", type=str, help="MISRA rules to suppress (comma-separated)")
+parser.add_argument("-verify", help="Verify check", action="store_true")
```

Comment:
I would like to hide `-verify` so it's not official. It is not intended for users.

Maybe hide `-generate-table` also. It was not intended that users would run this locally. The output is meant to be shown on the webpage not in the console. Do you think people wants to run it locally?

Reply:
It's an easy change to hide the parameters. Will do.